### PR TITLE
Temporary disable hit testing for ART Surface (iOS)

### PR DESCRIPTION
**Pull request by MaxGraey**
Temporary disable hit testing for ReactNativeART module until provide full events system for art elements. If we cover ART's Surface over ordinary view it will block any events, so we prevent this behaviour. 

@janicduplessis

### CODE REVIEW

File: Libraries/ART/ARTSurfaceView.m [hunk 1]
```diff
@@ -51,4 +51,8 @@ - (void)reactSetInheritedBackgroundColor:(UIColor *)inheritedBackgroundColor
   self.backgroundColor = inheritedBackgroundColor;
 }
 
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
+    return NO;
```
Comment (shergin) [inline Libraries/ART/ARTSurfaceView.m]: It should be `nil` instead of `NO`.

**Comment by MaxGraey** [comment]
Hi @shergin and thanks for reviewing. I don't remember why but self.userInteractionEnabled not works as expected in that time. It was approximately 5 month ago and maybe the problem is already solved. The main reason I used it - I needed to draw the custom graphic border on top of React element but ARTSurfaceView blocked any events passed to element

**Comment by shergin** [comment]
@MaxGraey Yeah, 5 months is a long time... 😞 
I think same thing can be achieved just by setting `pointerEvents = 'none'` on JS side, and it is probably right thing to do in your case.

Are you okay with closing this issue?

**Comment by MaxGraey** [comment]
Yes. I close this by myself) Thanks for response!

**Pull request closed without merging**